### DISCUSSION TITLE
Make Views foldable again

### DISCUSSION THREAD

**Discussion by onlinemumie** [standardnotes/forum#2783] [Suggestions]
There have been some major improvements in the last versions of Standard Notes (3.10.0 and on). What I really miss though is the option that was present previously, when it was possible to fold Views (with All notes, Archived, Trash and smart tags) on the desktop and web versions, like it still is on Android (3.11.0). Once you learn to code smart tags, the list tends to get long, and it is easier to just fold it instead of scrolling past it to reach the regular tags.

Also, if it at all possble, it would be great if smart tags one day could be folded in groups, like regular tags. Some of them tend to be for special searches only, invaluable but used used rarely.

**Comment by stefan-yas** [comment]
Good point, have raised it with the team. Thanks!

**Comment by kilger** [comment]
Seconded on these points.
